1.0/docs/devguide/templates.md

```diff
@@ -303,6 +303,30 @@ If you or a third-party library mutate the array without Polymer's methods,
 you need to call [`notifySplices`](#notifysplices) to ensure that any elements
 watching the array are properly notified.
 
+### Performance properties {#dom-repeat-perf}
+
+Use the following properties to affect the performance of `dom-repeat` 
+templates. 
+
+* [`delay`](/1.0/api/#dom-repeat:property-delay). Set a debounce time 
+  that must pass before a `filter` or `sort` function is re-run.
+* [`initialCount`](/1.0/api/#dom-repeat:property-initialCount). Define an 
+  initial number of template instances to render after setting the `items`
+  array, and put the `dom-repeat` template into chunking mode. The remaining
+  items are created and rendered incrementally at each animation frame until
+  all instances have been rendered.
+* [`renderedItemCount`](/1.0/api/#dom-repeat:property-renderedItemCount). The
```

Comment:
This doesn't belong in this list: you can't set it and it doesn't affect performance. It's a read-only property that tells you how many items the dom-repeat has rendered so far.